src/saml.ts

```diff
@@ -0,0 +1,3 @@
+export type SAML = {
```

Comment:
shouldn't this just be `preferredSingleSignOnMode` ? and then have a type of `saml` to signify only saml is supported for now?